corpus/convert.py

```diff
@@ -172,6 +189,36 @@ def run(self):
                     else segment.recording.name
                 )
                 segment_track = segment.track + 1 if segment.track else 1
+
+                orth = f" {segment.orth.strip()} "
+
+                if self.exclude_non_speech:
+                    for nst in self.non_speech_tokens:
+
+                        def replace_recursive(orthography, token):
```

Comment:
can you change this to a classmethod/staticmethod? A definition in a loop does not seem nice.